### PR TITLE
revised:L1 r-novice-gapminder

### PR DESCRIPTION
**Pull request by rkkmk**
https://github.com/swcarpentry-ja/i18n/commits?author=rkkmk

### CODE REVIEW

File: po/r-novice-gapminder.ja.po [hunk 1]
```diff
@@ -874,51 +874,50 @@ msgid ""
...
-"Science is a multi-step process: once you've designed an experiment and collected\n"
-"data, the real fun begins! This lesson will teach you how to start this process using\n"
-"R and RStudio. We will begin with raw data, perform exploratory analyses, and learn\n"
-"how to plot results graphically. This example starts with a dataset from\n"
-"[gapminder.org](https://www.gapminder.org) containing population information for many\n"
-"countries through time. Can you read the data into R? Can you plot the population for\n"
-"Senegal? Can you calculate the average income for countries on the continent of Asia?\n"
-"By the end of these lessons you will be able to do things like plot the populations\n"
-"for all of these countries in under a minute!"
+"科学は複数の段階からなる作業です：実験を計画したら、データを収集し、\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]: The original text implies three linear steps: plan the experiment, collect the data, do the analysis. I think we want to emphasize that the first two steps are done. How about:
```suggestion
"科学は複数の段階からなる作業です：実験を計画し、データを収集したら、\n"
```

File: po/r-novice-gapminder.ja.po [hunk 2]
```diff
@@ -927,63 +926,61 @@ msgid ""
 "on servers) and provides many advantages such as integration with version\n"
 "control and project management."
 msgstr ""
-"We'll be using RStudio: a free, open source R integrated development\n"
-"environment. It provides a built in editor, works on all platforms (including\n"
-"on servers) and provides many advantages such as integration with version\n"
-"control and project management."
+"このレッスンではRStudioを使います。RStudioは、無料であり、Rを組み込んだオープンソースの\n"
+"開発環境です。RStudioには、全てのプラットフォーム（サーバーも含む）で起動できるエディタ\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po:930]: The English distinguishes between including an editor and being able to run on all platforms. We may also want to add the definition of IDE (even though this is lacking in the English version).
```suggestion
"開発環境です。RStudioは、全てのプラットフォーム（サーバーも含む）で起動できること、\n"
```

File: po/r-novice-gapminder.ja.po [hunk 3]
```diff
@@ -927,63 +926,61 @@ msgid ""
...
+"が入っていて、プロジェクト管理やバージョン管理にも対応しているなど、良いところが\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"エディタが組み込まれていることや、プロジェクト管理やバージョン管理にも対応しているなど、\n"
```

File: po/r-novice-gapminder.ja.po [hunk 4]
```diff
@@ -927,63 +926,61 @@ msgid ""
...
+"いっぱいがあります。"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"良いところがいっぱいあります。"
```

File: po/r-novice-gapminder.ja.po [hunk 5]
```diff
@@ -997,20 +994,18 @@ msgid ""
...
-"to push the current line, selected lines or modified lines to the\n"
-"interactive R console.\n"
-"   * This is a great way to start; all your code is saved for later\n"
-"   * You will be able to run the file you create from within RStudio\n"
-"   or using R's `source()`  function."
+"後で走らせる.Rファイルにコピーする。\n"
+"   *  まず始めてみるときや、小規模なテストをするのに適している。\n"
+"   *  すぐ面倒になる。\n"
+"2. まず.Rファイルに書き、インタラクティブなRコンソールへ、現在の行、選択した行、\n"
+"修正した行を送るコマンドを走らせるために、RStudioのショートカットキーを使う。\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]:
```suggestion
"修正した行など送るコマンドを走らせるために、RStudioのショートカットキーを使う。\n"
```

File: po/r-novice-gapminder.ja.po [hunk 6]
```diff
@@ -927,63 +926,61 @@ msgid ""
...
 #: r-novice-gapminder/_episodes/01-rstudio-intro.md:73
 msgid "  * Environment/History (tabbed in upper right)"
-msgstr "  * Environment/History (tabbed in upper right)"
+msgstr "  * 環境／履歴（右上のタブ形式）"
 
 # unordered list
 #: r-novice-gapminder/_episodes/01-rstudio-intro.md:74
 msgid "  * Files/Plots/Packages/Help/Viewer (tabbed in lower right)"
-msgstr "  * Files/Plots/Packages/Help/Viewer (tabbed in lower right)"
+msgstr "  * ファイル／プロット／パッケージ／ヘルプ／ビューワー（右下のタブ形式）"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po:955]: Do these menu items appear in English or Japanese when RStudio is run on a computer with region set to Japan? If English, I think we should leave them untranslated (but include explanations of what they mean in parentheses).

File: po/r-novice-gapminder.ja.po [hunk 7]
```diff
@@ -1670,16 +1663,14 @@ msgid ""
 "And R will print out the answer, with a preceding \"[1]\". Don't worry about this\n"
 "for now, we'll explain that later. For now think of it as indicating output."
 msgstr ""
-"And R will print out the answer, with a preceding \"[1]\". Don't worry about this\n"
-"for now, we'll explain that later. For now think of it as indicating output."
+"するとRは、「[1]」の後に回答を表示します。詳しくは後で説明するので、\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po:1666]: We should clarify that "this" refers to the [1], not the rest of the output.
```suggestion
"するとRは、「[1]」の後に回答を表示します。「[1]」については後で説明するので、\n"
```

File: po/r-novice-gapminder.ja.po [hunk 8]
```diff
@@ -1874,13 +1864,12 @@ msgid ""
 "\"comment\". Anything that follows after the hash (or octothorpe) symbol\n"
 "`#` is ignored by R when it executes code."
 msgstr ""
-"The text after each line of code is called a\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po:1877]: This is something I think needs to be corrected in the original. The only thing that defines comments is that they come after the hashmark (on the same line), not placement "before" or "after" code.

File: po/r-novice-gapminder.ja.po [hunk 9]
```diff
@@ -2198,31 +2184,27 @@ msgid ""
...
+"> 以外では、２数の比較に決して `==` を使わないこと。\n"
 ">\n"
-"> Computers may only represent decimal numbers with a\n"
-"> certain degree of precision, so two numbers which look\n"
-"> the same when printed out by R, may actually have\n"
-"> different underlying representations and therefore be\n"
-"> different by a small margin of error (called Machine\n"
-"> numeric tolerance).\n"
+"> コンピュータは、小数点以下の数を限られた精度でしか示せない\n"
+"> かもしれません。すると、二つの数がRで同じように表示されても\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po:2191]:
```suggestion
"> なので、二つの数がRで同じように表示されても\n"
```

File: po/r-novice-gapminder.ja.po [hunk 10]
```diff
@@ -2198,31 +2184,27 @@ msgid ""
...
+"> 数を比較するときの注意： 整数型（整数のみを表すデータ型）\n"
+"> 以外では、２数の比較に決して `==` を使わないこと。\n"
 ">\n"
-"> Computers may only represent decimal numbers with a\n"
-"> certain degree of precision, so two numbers which look\n"
-"> the same when printed out by R, may actually have\n"
-"> different underlying representations and therefore be\n"
-"> different by a small margin of error (called Machine\n"
-"> numeric tolerance).\n"
+"> コンピュータは、小数点以下の数を限られた精度でしか示せない\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]: The "may" here means "be able to", not "maybe"
```suggestion
"> コンピュータは、小数点以下の数を限られた精度でしか示せません。\n"
```

File: po/r-novice-gapminder.ja.po [hunk 11]
```diff
@@ -2652,10 +2635,9 @@ msgid ""
 "mathematics, a vector in R describes a set of values in a certain order of the \n"
 "same data type. For example"
 msgstr ""
-"One final thing to be aware of is that R is *vectorized*, meaning that\n"
-"variables and functions can have vectors as values. In contrast to physics and\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po:2656]: Yet another issue with the original English. I wouldn't say functions can have vectors as values. Rather, (if they are vectorized) they can take vectors as arguments.

File: po/r-novice-gapminder.ja.po [hunk 12]
```diff
@@ -2652,10 +2635,9 @@ msgid ""
...
-"mathematics, a vector in R describes a set of values in a certain order of the \n"
-"same data type. For example"
+"留意すべきことの最後のひとつは、 Rは*ベクトル化している*ということで、つまり、\n"
+"変数と関数が値をベクトルとすることができます。 物理や数学とは対照的に、\n"
+"Rのベクトルは、同じデータ型のある順番を持つ値の集合として示されます。例えば："
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]: "describes" here means "defined as", not "shows".
```suggestion
"Rのベクトルは、同じデータ型のある順番を持つ値の集合として定義されています。例えば："
```

File: po/r-novice-gapminder.ja.po [hunk 13]
```diff
@@ -3015,15 +2991,14 @@ msgid ""
 "are over 10,000 packages available on CRAN (the comprehensive R archive\n"
 "network). R and RStudio have functionality for managing packages:"
 msgstr ""
-"It is possible to add functions to R by writing a package, or by\n"
-"obtaining a package written by someone else. As of this writing, there\n"
-"are over 10,000 packages available on CRAN (the comprehensive R archive\n"
-"network). R and RStudio have functionality for managing packages:"
+"パッケージを書くか、誰かが書いたパッケージを使って、Rに関数を加えることができます。\n"
+"これについて、CRAN (the comprehensive R archive network)上にある\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]: "As of this writing" means "currently"
```suggestion
"今現在CRAN (the comprehensive R archive network)上にある\n"
```

File: po/r-novice-gapminder.ja.po [hunk 14]
```diff
@@ -3035,11 +3010,11 @@ msgid ""
 "* You can make a package available for use with `library(packagename)`"
 msgstr ""
 "  `installed.packages()`\n"
-"* You can install packages by typing `install.packages(\"packagename\")`,\n"
-"  where `packagename` is the package name, in quotes.\n"
-"* You can update installed packages by typing `update.packages()`\n"
-"* You can remove a package with `remove.packages(\"packagename\")`\n"
-"* You can make a package available for use with `library(packagename)`"
+"* パッケージをインストールするには、`install.packages(\"packagename\")`を入力します\n"
+"  ここで、`packagename`は、用いるパッケージ名です。\n"
```
Comment (joelnitta) [inline po/r-novice-gapminder.ja.po]: Important to mention including quotes around the package name here. The fact that it's not needed for `library()` is probably something that should be clarified in the original lesson.
```suggestion
"  ここで、`packagename`は、用いる引用符付きのパッケージ名です。\n"
```

**Review by joelnitta — CHANGES_REQUESTED** [review]
Thanks! This looks great. I found some spots where I think the translation needs fixing - please see my suggestions. In addition I found a few issues that need to be corrected in the main DC/SWC repo, but noting them here first.

File: po/r-novice-gapminder.ja.po (hunk 11 above)
Comment (rkkmk) [reply po/r-novice-gapminder.ja.po:2656]: I thought this description is related to the fact that "R has no 0-dimensional, or scalar types. Individual numbers or strings, which you might think would be scalars, are actually vectors of length one."
(quoted from http://adv-r.had.co.nz/Data-structures.html)

File: po/r-novice-gapminder.ja.po (hunk 2 above)
Comment (rkkmk) [reply po/r-novice-gapminder.ja.po:930]: How about adding IDE next to the word? Then, the reader got interested in can look it up online. (I couldn't come up with a concise definition which wouldn't make the sentence too long...) 

```suggestion
"総合開発環境（IDE: Integrated Development Environment）です。RStudioは、全てのプラットフォーム（サーバーも含む）で起動できること、\n"
```

File: po/r-novice-gapminder.ja.po (hunk 4 above)
Comment (rkkmk) [reply po/r-novice-gapminder.ja.po]: Are you suggesting we should remove 良いところが from line 931 and move to 932??

File: po/r-novice-gapminder.ja.po (hunk 10 above)
Comment (rkkmk) [reply po/r-novice-gapminder.ja.po]: I agree with your suggestion. Then かもしれません。on the line 2191 should also be removed at the same time?

File: po/r-novice-gapminder.ja.po (hunk 11 above)
Comment (joelnitta) [reply po/r-novice-gapminder.ja.po:2656]: Actually, I don't think that quote from Advanced R directly applies here. The problem is that the original version conflates arguments to functions with functions themselves. At any rate, let's leave the translation as it is, and file an issue to fix this in the main SWC lesson repo.

**Review by joelnitta — APPROVED** [review]

**Comment by TomKellyGenetics** [comment]
These changes have been added to the repository :-)

**Pull request merged**